opennms-wmi/src/main/java/org/opennms/netmgt/collectd/WmiCollector.java

```diff
@@ -114,6 +118,8 @@ public void initialize() {
         runtimeAttributes.put(WMI_COLLECTION_KEY, collection);
         final WmiAgentConfig agentConfig = WmiPeerFactory.getInstance().getAgentConfig(agent.getAddress());
         runtimeAttributes.put(WMI_AGENT_CONFIG_KEY, agentConfig);
+        final WmiResourceTypeListWrapper wmiResourceTypeListWrapper = new WmiResourceTypeListWrapper(DataCollectionConfigFactory.getInstance().getConfiguredResourceTypes());
+        runtimeAttributes.put(WMI_RESOURCE_TYPES_KEY, wmiResourceTypeListWrapper);
```

Comment:
This results in *all* the resource types being sent to the Minion on each request for collection - which isn't ideal